Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity

```diff
@@ -54,14 +54,15 @@ LightmapSettings:
     m_EnableBakedLightmaps: 1
     m_EnableRealtimeLightmaps: 0
   m_LightmapEditorSettings:
-    serializedVersion: 10
+    serializedVersion: 12
```

Comment:
It looks like there are a lot of serialization changes in here. Would it be possible for the logo updates to be made in Unity 2018?

Reply:
Oh, I see. I'll use 2018.